packages/commerce-sdk-react/src/hooks/ShopperBaskets/mutation.ts

```diff
@@ -373,8 +391,8 @@ export const getCacheUpdateMatrix = (customerId: string | null) => {
             const basketId = params.parameters.basketId
 
             return {
-                ...updateBasketQuery(basketId),
-                ...invalidateCustomerBasketsQuery(customerId)
+                ...updateBasketQuery(basketId, response)
+                // ...invalidateCustomerBasketsQuery(customerId)
```

Comment:
Is there a conversation we need to have about this? or did you simply forget to remove that commented line?

Reply:
We don't need it here, I've removed it